netket/models/DeepSets.py

```diff
@@ -0,0 +1,170 @@
+from typing import Union, Tuple, Any
```

Comment:
I would not include the DeepSet for now, since it is still under investigation/ we don't have a standard model for this yet

Reply:
I do agree with @gcarleo but if we merge this can we have at least a toy model that works for a toy problem?

Reply:
So that it is a bit more clear for whoever decides to play with this what the model should do.

Reply:
yes I would just do the harmonic oscillator with a gaussian ansatz in the notebook, no deepset involved...